dev/tools/make-changelog.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+
+echo "PR number"
+read -r PR
+
+echo "Where? (type a prefix)"
+ls doc/changelog
+read -r where
+
+where=$(echo doc/changelog/"$where"*)
+where="$where/$PR-$(git rev-parse --abbrev-ref HEAD).rst"
+
+# shellcheck disable=SC2016
+# the ` are regular strings, this is intended
+# use %s for the leading - to avoid looking like an option (not sure
+# if necessary but doesn't hurt)
+printf '%s bla bla (`#%s <https://github.com/coq/coq/pull/%s>`_, by %s).' - "$PR" "$PR" "$(git config user.name)" > "$where"
+
+$(git config core.editor) "$where" || echo "$where"
```

Comment:
`git config core.editor` returns an empty value for me (you could use the `$EDITOR` environment variable which is defined for me as a fallback). In any case, this line is not very robust to `git config core.editor` returning an empty value: the fact that you get a strange error message is confusing.